src/components/Header/Header.js

```diff
@@ -46,23 +46,34 @@ type Props = {
   flexStart?: boolean,
   light?: boolean,
   style?: Object,
-  headerRightFlex?: string,
-  overlay?: boolean,
+  headerRightFlex?: number,
   backIcon?: string,
   nextIconSize?: number,
   titleStyles?: ?Object,
+  headerRightAddon?: React.Node,
+  white?: boolean,
 }
 
 const Wrapper = styled.View`
   border-bottom-width: 0;
-  padding: ${props => props.noPadding ? 0 : '0 20px'};
-  height: ${({ noWrapTitle }) => noWrapTitle ? 'auto' : '48px'};
+  padding: ${props => props.noPadding ? 0 : '0 16px'};
+  z-index: 10;
+  ${props => props.white
+    ? `
+      background-color: ${baseColors.white};
+      border-bottom-width: 1px;
+      border-bottom-color: ${baseColors.mediumLightGray};
+    `
+    : ''}
+`;
+
+const InnerWrapper = styled.View`
   justify-content: flex-end;
   align-items: flex-end;
   flex-direction: row;
   margin-top: ${spacing.rhythm};
-  margin-bottom: ${props => props.flexStart ? 'auto' : 0};
-  z-index: 10;
+  margin-bottom: ${props => props.flexStart ? 'auto' : 4};
```

Comment:
Shouldn't it be '4px'?

Reply:
Yeah, it would be safer to add unit. I'll fix this